### ISSUE TITLE
When will omniedge-ios  be available on AppStore? 

### ISSUE THREAD

**Issue by Creling** [omniedgeio/omniedge-ios#1]
Hi, glad to see omniedge-ios is open-source now, but it is no available on AppStore so I have no way install it. when will it be available on AppStore?

**Comment by zteshadow** [comment]
Yeah, we are working on a new version for AppStore right now and will come out in about 2 months
